tests/default.nix

```diff
@@ -93,6 +93,7 @@ import nmt {
     ./modules/programs/ne
     ./modules/programs/neomutt
     ./modules/programs/newsboat
+    ./modules/programs/nheko
```

Comment:
> `nheko` seems to be marked as broken for MacOS, anything I should do here?

The package needs to be stubbed in the tests.

Reply:
Ah that's what those stubs were for! I had them before but removed them because the tests were passing anyway, but of course, I'm on NixOS not on Mac 😄